src/utils/objectManipulation.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * For setting and manipulating object
+ *     values in general.
+ */
+
+// I can't figure out how to use lodash's
+// `cloneDeep`, so I'm resorting to this.
+const cloneDeep = function ( obj ) {
+  return JSON.parse( JSON.stringify( obj ) );
+};
+
+
+export {
+  cloneDeep
+};
```

Comment:
If you're using lodash's `cloneDeep`, can this be safely deleted?

Reply:
This can! I thought I did! Thanks for picking up on that.